docs/buildah-bud.md

```diff
@@ -733,9 +737,10 @@ are stored while pulling and pushing images.  Defaults to '/var/tmp'.
 
 ### `.dockerignore`
 
-If the file .dockerignore exists in the context directory, `buildah bud` reads
-its contents. Buildah uses the content to exclude files and directories from
-the context directory, when executing COPY and ADD directives in the
+If the file .dockerignore exists in the context directory, `buildah copy` reads
```

Comment:
There's no context directory for `buildah copy`, same as with `buildah add`.

Reply:
Yes, but this is the buildah bud man page.  The copy and add man pages point out that the --contextdir option is required to use --ignorefile.